### DISCUSSION TITLE
OpsAdministration---> alex---> dev---> Core---> PostInstallScript

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#533] [Bugs] [answered]
HI Team , 
we are getting an error , while running the PostInstallScript jenkins job , below we have provided jenkins console.

TASK [post-install : uri] ******************************************************
fatal: [localhost]: FAILED! => {
    "access_control_allow_origin": "*",
    "changed": false,
    "connection": "close",
    "content": "{\"id\":\"api.org.create\",\"ver\":\"v1\",\"ts\":\"2023-06-08 04:46:58:460+0000\",\"params\":{\"resmsgid\":\"59ce303c231130d045a2c824a8cea9bd\",\"msgid\":\"59ce303c231130d045a2c824a8cea9bd\",\"err\":\"UOS_ORGCRT0004\",\"status\":\"FAILED\",\"errmsg\":\"Value sunbird for channel is already in use.\"},\"responseCode\":\"CLIENT_ERROR\",\"result\":{}}",
    "content_length": "309",
    "content_type": "application/json",
    "date": "Thu, 08 Jun 2023 04:46:58 GMT",
    "elapsed": 0,
    "json": {
        "id": "api.org.create",
        "params": {
            "err": "UOS_ORGCRT0004",
            "errmsg": "Value sunbird for channel is already in use.",
            "msgid": "59ce303c231130d045a2c824a8cea9bd",
            "resmsgid": "59ce303c231130d045a2c824a8cea9bd",
            "status": "FAILED"
        },
        "responseCode": "CLIENT_ERROR",
        "result": {},
        "ts": "2023-06-08 04:46:58:460+0000",
        "ver": "v1"
    },
    "redirected": false,
    "referrer_policy": "origin-when-cross-origin, strict-origin-when-cross-origin",
    "status": 400,
    "url": "https://alex-sb.dev.verlab.io/api/org/v1/create",
    "via": "kong/0.14.1",
    "x_content_type_options": "nosniff",
    "x_frame_options": "DENY",
    "x_kong_proxy_latency": "7",
    "x_kong_upstream_latency": "23",
    "x_permitted_cross_domain_policies": "master-only",
    "x_ratelimit_limit_hour": "5000",
    "x_ratelimit_remaining_hour": "4999",
    "x_xss_protection": "1; mode=block"
}

MSG:

Status code was 400 and not [200]: HTTP Error 400: Bad Request

**Comment by chitranshu-keshav** [comment]
@santhosh-tg can you please get back on this

**Reply by tsprasath** [reply, marked as answer]
Issue is resolved as part of https://github.com/orgs/Sunbird-Ed/discussions/516
